lib/createLiveStyleApp.js

```diff
@@ -67,10 +70,14 @@ module.exports = function createLiveStyleApp(options) {
 
     if (options.root) {
         if (options.compiless) {
+            compiless = require('express-compiless');
+
             app.use(compiless({root: options.root}));
```

Comment:
You could save the var and just go `app.use(require('express-compiless')({root: options.root}));`